Docs - Develop - Organize examples (#1228)
iotaledger/iota.rs: documentation/docs/_admonitions/_password_storage.md

:::warning Password Storage

> It is not recommended to store passwords or seeds on a host's environment variables or in the source code in a production

Reviewer: ```suggestion
Never store passwords or seeds on a host's environment variables or in the source code in a production
```
